lib/global_dialog.dart

```diff
@@ -128,13 +134,16 @@ class _GlobalDialogState extends State<GlobalDialog> {
   }
 
   void showLoading(BuildContext context,
-      {required bool loading, bool isRootOverlay = true}) {
+      {required bool loading,
+      required bool isRootOverlay,
+      required bool blockGestures}) {
```

Comment:
Here `isRootOverlay` & `blockGestures` is required but within the static function.. please make it optional similar to static function